README.md

```diff
@@ -132,31 +81,67 @@ output = scenario.simulate(object_path="vehicle.obj",
                            resolution = "medium")
 ```
 
-The user needs to specify the object to insert inside the wind tunnel and thereafter,
-select the total simulation time, the step between outputs and the resolution of the simulation.
+The user must provide the path for the object to be inserted in the wind tunnel. Additionally, users can choose the total simulation time, the time step interval to output data and the resolution of the simulation.
 
-After the simulation has finished, the user has the possibility to obtain several metrics:
+After the simulation has finished, the user can obtain several metrics and visuals. An example here is given for a slice of the flow:
 
 ```python
 
-pressure_field = output.get_physical_field("pressure")
+flow_slice = output.get_flow_slice(simulation_time=100,
+                                   plane="xz")
+flow_slice.render("velocity")
+```
+
+![F1 car in WindTunnel](resources/media/wind_tunnel.png)
+
+### Fluid tank
+
+This scenario simulates the motion of a fluid in a cubic or cylindrical tank.
+Fluid is injected into the tank via an inlet located at the top of the tank and
+flows out of the tank via an outlet located at the bottom of the tank. The
+motion of the fluid is controlled by gravity. The simulation is performed using
+the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
+method.
+
+#### Example
+
+Initialize the scenario:
+
+```python
+from inductiva import fluids
+scenario = fluids.scenarios.FluidTank(shape=fluids.shapes.Cube(),
+                                      fluid=fluids.WATER,
+                                      fluid_level=0.5)
 ```
 
-Visualize the outputs:
+The user can specify the fluid (e.g. water, honey or oil), the fluid level (in
+meters), as well as the tank shape and its inlet and outlet properties.
+
+Run the simulation:
 
 ```python
-pressure_field.render()
+output = scenario.simulate(simulation_time=5,
+                           output_time_step=0.1,
+                           resolution="medium")
 ```
 
-\[TODO @IvanPombo: add gif with render\]
+The user can specify the total simulation time and the time step between outputs
+(all in seconds). The user can also specify the resolution of the simulation
+(low, medium or high).
+
+Visualize the results:
+
+```python
+output.render()
+```
 
+![Fluid tank simulation.](resources/media/fluid_tank.gif)
 
 ### Coastal area
 
 This scenario simulates the propagation of waves in a coastal area, by solving
 the [shallow water equations](https://en.wikipedia.org/wiki/Shallow_water_equations)
-in a given bathymetric profile (i.e., the depth of the sea bottom). For now,
-this profile is fixed to be that of Praia do Carneiro beach, in Porto, Portugal.
+in a given bathymetric profile (i.e., the depth of the sea bottom). 
```

Comment:
Please do not remove the bit about Praia do Carneiro yet. This still uses a fixed bathymetry.

Reply:
I know, but in any case it is preferable to not have it here, since it limits the scope of the scenario. Even if we just add it a bit after.

Reply:
I think the README should be source of truth, and that truth will only change later today/tomorrow when we add support to read bathymetries from files! Proceed as you wish, we can adapt this later.